nexuse2e-core/src/main/java/org/nexuse2e/util/CertificateUtil.java

```diff
@@ -490,6 +490,19 @@ public static String getRemainingValidity(X509Certificate x509certificate) {
         return remaining;
     } // getRemainingValidity
 
+    /**
+     * @param x509certificate
+     * @return int RemainingDayCount
+     */
+    public static int getRemainingDayCount(X509Certificate x509certificate) {
+        long nowMils = System.currentTimeMillis();
+        long expMils = x509certificate.getNotAfter().getTime();
```

Comment:
Are both values UTC or do we have problems with timezone near the date barrier?

Reply:
No problems.